transport/http/server/server.go

```diff
@@ -152,34 +162,16 @@ func ParseTLSConfigWithLogger(cfg *config.TLS, logger logging.Logger) *tls.Confi
 	}
 
 	tlsConfig := &tls.Config{
-		MinVersion:               parseTLSVersion(cfg.MinVersion),
-		MaxVersion:               parseTLSVersion(cfg.MaxVersion),
-		CurvePreferences:         parseCurveIDs(cfg),
-		PreferServerCipherSuites: cfg.PreferServerCipherSuites,
```

Comment:
we do not use the value provided by the config anymore because it has no effect (is deprecated)